<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>821</width>
    <height>614</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="login_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="user_edit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>SSSSSSFASFASDFASDFASD</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>用户名：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pwd_edit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>SSSSSSFASFASDFASDFASD</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>密码：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clear_button">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>180</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>清除</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>20</y>
     <width>291</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>300</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
